### ISSUE TITLE
0.1.1 was broken `php vendor/preload.php` command

### ISSUE THREAD

**Issue by chrisLeeTW** [Ayesh/Composer-Preload#9]
due to this line : https://github.com/Ayesh/Composer-Preload/blob/v0.1.1/src/PreloadWriter.php#L74

should be
```
if (!\function_exists('opcache_compile_file') || !\ini_get('opcache.enable')) {
```

**Comment by Ayesh** [comment]
Thanks for opening this issue @chrisLeeTW - Do you get any error messages when you try to load the file? Those backslashes were added for a micro-optimization with fully-qualified function names. However, `\f` is a special character, so that might be the problem. Knowing the error message could help a lot :)

**Comment by linaspasv** [comment]
@Ayesh it's not about slashes but if logic is completely wrong now. You might have deleted not directive by accident for function_exists('opcache_compile_file').

**Comment by Ayesh** [comment]
Thanks @linaspasv - you are right, there indeed was a missing negation in the first `if` expression. I have released a hotfix v0.1.2 and also re-tagged v0.1.1. Thank you.
